next.json

```diff
@@ -414,9 +438,84 @@
             }
         ]
     },
+    "depth stencil state": {
+        "category": "object"
+    },
+    "depth stencil state builder": {
+        "category": "object",
+        "methods": [
+            {
+                "name": "get result",
+                "returns": "depth stencil state"
+            },
+            {
+                "name": "set depth enabled",
```

Comment:
nit: I kind of like the Metal approach of having this be implicitly defined by whether the compare function is "always" or not. What do you think?